### DISCUSSION TITLE
Welcome to laravel-gedcom Discussions!

### DISCUSSION THREAD

**Discussion by curtisdelicata** [liberu-genealogy/laravel-gedcom#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CODEMAX666** [comment]
Hello.
I am a software engineer and I want to contribute to open source with skill👋. 😀

**Reply by curtisdelicata** [reply]
Please contact us on genealogysoftwareuk@gmail.com

**Reply by curtisdelicata** [reply]
Feel free to send pull requests to be considered on any repository

**Comment by curtisdelicata** [comment]
@CODEMAX666

**Reply by CODEMAX666** [reply]
Thank you.
